taxonomy/utils.py

```diff
@@ -207,3 +207,23 @@ def get_blacklisted_course_skills(course_key, prefetch_skills=True):
     if prefetch_skills:
         qs = qs.select_related('skill')
     return qs.all()
+
+
+def get_course_jobs(course_key):
+    """
+    Get all the course jobs.
+
+    Arguments:
+        course_key (str): Key of the course whose course skills need to be returned.
+
+    Returns:
+        list: A list of all the course jobs.
+    """
+    course_skills = get_whitelisted_course_skills(course_key)
+    job_skills = JobSkills.objects.select_related(
+        'skill',
+        'job',
+    ).filter(
+        skill__in=[course_skill.skill for course_skill in course_skills]
+    )
+    return list({job_skill.job.name for job_skill in job_skills})
```

Comment:
why create a set and the convert it to list?  you can either return the set or create a list in the first place.
```
return {job_skill.job.name for job_skill in job_skills}
# OR
return [job_skill.job.name for job_skill in job_skills]
```